test/langPickerTest.js

```diff
@@ -2,33 +2,158 @@
 
 const assert = require('assert');
 const LanguagePicker = require('../lib/LanguagePicker');
+const defaultMap = require('../lib/fallbacks.json');
```

Comment:
I think it would be better to have the tests depend on a small hardcoded languageMap rather than load and depend on the actual fallbacks file. These can change and be externalized but the logic here shouldn't be affected.

Reply:
Fair enough. I'll add segments to the tests that are relevant.